boards/arm/da1469x_dk_pro/Kconfig.defconfig

```diff
@@ -8,4 +8,11 @@ if BOARD_DA1469X_DK_PRO
 config BOARD
 	default "da1469x_dk_pro"
 
+if LVGL
+
+config LV_Z_AREA_X_ALIGNMENT_WIDTH
```

Comment:
A depends on makes this smaller, if you only have one option.

Reply:
Done